fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpapi/media/BaseWPV2MediaRestClient.kt

```
@@ -123,10 +121,10 @@ class WPV2MediaRestClient @Inject constructor(
             }
 
             val request = Request.Builder()
-                    .url(url)
-                    .post(body = body)
-                    .header(WPComGsonRequest.REST_AUTHORIZATION_HEADER, "Bearer ${accessToken.get()}")
-                    .build()
+                .url(url)
+                .post(body = body)
+                .header(WPComGsonRequest.REST_AUTHORIZATION_HEADER, getAuthorizationHeader(site))
```

In `ApplicationPasswordsMediaRestClient`'s implementation, this might return an empty string which I suppose will make the request fail.

Do we want to fail early here in that case? Or keep using the existing error handling below?

---

Good point, we can do so, but I didn't want to complicate logic for an unlikely scenario: the media feature is deep in all of the clients, the user won't reach it until they have a valid application password, so the chances of having an empty string here should be very minimal.
Since the result is the same, just would take a slightly longer, I opted for code simplicity.